udata/tests/test_transfer.py

```diff
@@ -8,6 +8,8 @@
     transfer_request_notifications
 )
 from udata.models import Member
+from udata.core.user.metrics import update_owner_metrics
+from udata.core.organization.metrics import update_owner_metrics
```

Comment:
I don't think you're supposed to include the same function name twice 😛 

Maybe the second one should be `update_organizaton_metrics`?